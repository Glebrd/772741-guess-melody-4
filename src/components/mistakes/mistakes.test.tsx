import * as React from "react";
import * as renderer from "react-test-renderer";
import Mistakes from "./mistakes";

describe(`Mistakes should render corrrectly`, () => {
  it(`With one zero count`, () => {
    const tree = renderer
      .create(<Mistakes
        count={0}
      />)
        .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Wit one one count`, () => {
    const tree = renderer
      .create(<Mistakes
        count={1}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
