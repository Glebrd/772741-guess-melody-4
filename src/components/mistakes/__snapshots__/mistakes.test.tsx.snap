// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes should render corrrectly Wit one one count 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;

exports[`Mistakes should render corrrectly With one zero count 1`] = `
<div
  className="game__mistakes"
/>
`;
